CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 5.2.0 - 2022-09-27
+
+* Add support for new security features when sending a file by email:
+  * `confirmEmailBeforeDownload` can be set to `true` to require the user to enter their email address before accessing the file.
+  * `retentionPeriod` can be set to `<1-78> weeks` to set how long the file should be made available.
```

Comment:
can you just tweak this to say that these (and isCsv) are now in an options object?

Reply:
yep yep 👍